### PR TITLE
Update docker secrets

### PR DESCRIPTION
**Pull request by alex-glv**
Docker secrets are added on organizational level, so we don't have to duplicate them, but they're named differently.

### CODE REVIEW

**Comment by andreas131989** [comment]
LGTM but are you sure these are the right creds?
The workflow fails to push on dockerhub with unauthorized

**Comment by alex-glv** [comment]
> LGTM but are you sure these are the right creds?
> The workflow fails to push on dockerhub with unauthorized

I fixed the permissions on docker hub, build passes now.

**Review by andreas131989 — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -22,6 +22,9 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM gcr.io/distroless/static:nonroot
+
+LABEL org.opencontainers.image.source="https://github.com/ContainerSolutions/redis-operator"
```
Comment (andreas131989) [inline Dockerfile:26]: I think you mean pre-scaling operator :D

Comment (containersolutions-int) [reply Dockerfile]: 🤦 Fixed it. Thank you.

**Pull request merged**